op-batcher/batcher/channel_builder_test.go

```diff
@@ -323,8 +323,12 @@ func FuzzSeqWindowClose(f *testing.F) {
 		// Check the timeout
 		cb.timeout = timeout
 		cb.updateSwTimeout(&derive.BatchData{
-			BatchV1: derive.BatchV1{
-				EpochNum: rollup.Epoch(epochNum),
+			BatchV2: derive.BatchV2{
+				Version:   2,
```

Comment:
Are we using version 0 for single batcher and version 1 for multiple batcher or 1 and 2 respectively as suggested in the code?

Reply:
The 0 is the serialization header byte... The numerical value is arbitrary, and we could use 0 when deserialized from `BatchV1` to conform to the serialization, or 1 to conform to the existing naming. I chose the naming so that an uninitialized BatchV2 would always be obvious in debugging, but it doesn't need to be like that.